src/algebra/lie/cartan_subalgebra.lean

```diff
@@ -0,0 +1,96 @@
+/-
+Copyright (c) 2021 Oliver Nash. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Oliver Nash
+-/
+import algebra.lie.nilpotent
+
+/-!
+# Cartan subalgebras
+
+Cartan subalgebras are one of the most important concepts in Lie theory. We define them here.
+The standard example is the set of diagonal matrices in the Lie algebra of matrices.
+
+## Main definitions
+
+  * `lie_subalgebra.normalizer`
+  * `lie_subalgebra.le_normalizer_of_exists_ideal`
+  * `lie_subalgebra.is_cartan_subalgebra`
+
+## Tags
+
+lie subalgebra, normalizer, idealizer, cartan subalgebra
```

Comment:
IMO Lie and Cartan should be uppercase here.

Reply:
I generally agree with capitalising names but I've been consistently sticking to pure lowercase for these `Tags` across 15+ files so I'm reluctant to make this file the odd one out!

Of course if someone wants to submit a PR capitalising all the names in `Tags` in `src/algebra/lie/*.lean` then that would be fine by me :-)